runtime/near-test-contracts/build.rs

```diff
@@ -0,0 +1,50 @@
+use std::process::Command;
+use std::{fs, io, process};
+
+fn main() {
+    if let Err(err) = try_main() {
+        eprintln!("{}", err);
+        process::exit(1);
+    }
+}
+
+fn try_main() -> io::Result<()> {
+    let status =
+        Command::new("rustup").args(&["target", "add", "wasm32-unknown-unknown"]).status()?;
+    if !status.success() {
+        return Err(io::Error::new(io::ErrorKind::Other, "rustup target add failed"));
+    }
+
+    build_contract("./test-contract-rs", &[], "test_contract_rs")?;
+    build_contract(
+        "./test-contract-rs",
+        &["--features", "nightly_protocol_features"],
+        "nightly_test_contract_rs",
+    )?;
+    build_contract("./tiny-contract-rs", &[], "tiny_contract_rs")?;
+    Ok(())
+}
+
+fn build_contract(dir: &str, args: &[&str], output: &str) -> io::Result<()> {
+    let mut cmd = cargo_build_cmd();
+    cmd.args(args);
+    cmd.current_dir(dir);
+    let status = cmd.status()?;
+    if !status.success() {
+        return Err(io::Error::new(io::ErrorKind::Other, "cargo build failed"));
+    }
+    fs::copy(
+        format!("./{}/target/wasm32-unknown-unknown/release/{}.wasm", dir, dir.replace('-', "_")),
+        format!("./res/{}.wasm", output),
+    )?;
+    println!("cargo:rerun-if-changed=./{}/src/lib.rs", dir);
+    println!("cargo:rerun-if-changed=./{}/Cargo.toml", dir);
```

Comment:
What are these println?

Reply:
They tell cargo when it needs to rebuild the test contracts, These are documented in https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

In theory, we maybe can just always run `cargo build` here, as that should be a no-op, but it wasn't hard to save an extra process invocation here.